client/jetpack-connect/main.jsx

```diff
@@ -332,7 +332,7 @@ export class JetpackConnectMain extends Component {
 			return {
 				headerTitle: translate( 'Get Jetpack Personal' ),
 				headerSubtitle: translate(
-					'Security essentials for every WordPress site ' +
+					'Security essentials for your WordPress site ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Security essentials for every WordPress site including automated backups and priority support.' )` *ES Score: 16*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
The purpose of this PR is to change the previous string to remove ambiguity as to whether a plan is required for each site. I'm afraid we can't use the existing translations.